lib/daru.rb

```diff
@@ -80,7 +80,7 @@ def plotting_library= lib
   create_has_library :gruff
 end
 
-[['reportbuilder', '~>1.4'], ['spreadsheet', '~>1.1.1']].each do |lib|
+[['spreadsheet', '~>1.1.1']].each do |lib|
```

Comment:
With only one library left, there is no need for this complicated machinery anymore ;)